tests/conftest.py

```diff
@@ -153,28 +150,13 @@ def execute_votes(accounts, vote_ids, dao_voting, topup="10 ether"):
             print(f"vote #{vote_id} executed")
             execution_transactions.append(tx)
 
-        Helpers._prefetch_contracts_from_etherscan()
-
         return execution_transactions
 
     @staticmethod
     def is_executed(vote_id, dao_voting):
         vote_status = dao_voting.getVote(vote_id)
         return vote_status[1]
 
-    @staticmethod
-    def _prefetch_contracts_from_etherscan():
```

Comment:
Is it certain that this method is no longer needed? In some cases, missing contract fetching has caused event validation failures.